### PR TITLE
feat: link to original image file

### PR DESCRIPTION
**Pull request by bedgarone**
## Description

The goal of this issue #194 is to provide a way for the user to access the _Capture_'s image in its original size, by opening it in a new tab or copying its url.

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
- Enhancement

## Issue
**What is the current behavior?**
Each capture row pops up an optimized image (resized to improve performance) and there isn't a path to reach the original image in its full size.  

**What is the new behavior?**
The user is now able to either copy the original image's url or open the image in a new tab. 

**Steps taken:**
- Added a new item to the _Capture Data_'s array, which adds another item (the original image url) in the right panel;
- Set that itemps 'copy' property to true, so that a copy item appears and enables the user to copy the url;
- Added the same link to the original image over the optimized image, with _absolute_ positioning.

### Screenshots
imagem
imagem
imagem

### CODE REVIEW

**Comment by bedgarone** [comment]
@nmcharlton these were simple implementations of the discussed ideas. I'm would need some opinion on the link over the image, isn't it too distracting? Perhaps the link in the right panel is sufficient? 
Let me know if any of these need changes. Also, if the link over the image is going ahead, some refactor is needed in the style as I merely used strings while I suspect some items should use theme values (like the color, etc.).

**Comment by nmcharlton** [comment]
@bedgarone This looks great! The link and copy option in the side panel as you've proposed work well.
We can remove the link from the image; I agree it's a bit distracting.

Nice work.

File: src/components/CaptureDetailDialog.js [hunk 1]
```diff
@@ -313,6 +340,14 @@ function CaptureDetailDialog(props) {
           style={{ maxWidth: '100%' }}
           fixed
         />
+        <Link
```
Comment (nmcharlton) [inline src/components/CaptureDetailDialog.js:343]: Let's remove this link from the image.

**Review by nmcharlton — CHANGES_REQUESTED** [review]

File: src/components/CaptureDetailDialog.js (hunk 1 above)
Comment (bedgarone) [reply src/components/CaptureDetailDialog.js]: Hey @nmcharlton, just removed that link.

**Review by nmcharlton — APPROVED** [review]
Nice work

**Pull request merged**